tutorial/4_hextof_workflow.ipynb

```diff
@@ -863,6 +863,26 @@
     "Once all the calibrations are done, a new run can be loaded by simply calling all the calibration functions."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sed.core.config import load_config\n",
+    "import numpy as np\n",
+    "metadata = load_config(path + \"/44824_20230324T060430.json\")\n",
```

Comment:
I have put the metadata file in the shared folder so this should work but is rather a crude way
`metadata = load_config(beamtime_dir + "/shared/44824_20230324T060430.json")`

Reply:
Included